STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,65 @@
+# Istio Steering Committee
+
+The Istio Steering Committee was formed to oversee the administrative aspects of the project, including governance, branding, and marketing. Steering was created to allow the Techincal Oversight Committee to exclusively focus on the technical aspects of the project. 
+
+## Steering Committee Charter
+
+1. Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Istio
+2. Charter and refine policy for community groups (including Working Groups and Committees)
+3. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
+4. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository
+5. Control and delegate access to and establish processes regarding other project resources/assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
+6. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
+7. Resolve any dispute from the Technical Oversight Committee.
```

Comment:
Maybe "Resolve any disputes escalated by the Technical Oversight Committee."?

Should we have a mechanism for people not agreeing with TOC and taking things to steering? Or do we want to funnel all through TOC?

What about non-technical disputes like code of conduct violations?

Reply:
I don't think we want Steering to override a TOC decision, do we? Since Steering may or may not have the right technical folks in it. I do think it makes sense that if TOC is unable to decide something, it kicks up to Steering. 

When the community doesn't agree with something TOC has decided; I see them raising that on Github or in working groups, etc. In those cases would it be best for the community and TOC to discuss to reach agreement? 

Code of conduct should go to Steering. I'll make that more clear in our existing CoC docs.

Reply:
OK I like that model, TOC is final arbiter of technical disputes, but TOC can kick things up to steering. I think we need to reword this section to make that clear.

Reply:
Is the goal here to have senate like rules, if the TOC is hung then steering gets to cast a deciding vote ?